OurUmbraco/Auth/UmbracoMemberAuthToken.cs

```diff
@@ -0,0 +1,146 @@
+﻿using System;
+using System.Net;
+using System.Net.Http;
+using System.Web.Http.Controllers;
+using System.Web.Http.Filters;
+using Umbraco.Core;
+using Umbraco.Core.Models;
+using Umbraco.Web;
+using JWT;
+
+namespace OurUmbraco.Auth
+{
+    public class UmbracoMemberAuthToken : ActionFilterAttribute
```

Comment:
Authorization and Authentication are 2 different things. This is combining both of them. Although that will work it's not the greatest way to do things. The authentication part should be in the very beginning of the request and add claims to the user/principal created in the request, the Authorization filter (this thing) should just be checking claims. This filter should also be an AuthorizationFilter not just an ActionFilter. 

I'm going to help review all of this and see if we can wrangle this in properly. Aspnet comes with all of the code and ability to Authenticate properly with jwt or bearer token authentication without having to re-invent the wheel since all of this code is manually re-doing what has already been done.